great-idea-website/index.html

```diff
@@ -1,6 +1,7 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
 
```

Comment:
You might want to add `<meta name="viewport" content="width=device-width, initial-scale=1.0">` here